types/sequelize/index.d.ts

```diff
@@ -5105,6 +5105,27 @@ declare namespace sequelize {
     interface Options {
 
         /**
+         * The username which is used to authenticate against the database.
```

Comment:
@aozgaa I know, but what I mean is that if we follow @SimonSchick 's suggestion, we need 3 definition of Options. And I suggest just add some property that have no use in some situation.